<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabPestanias">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>231</width>
      <height>291</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Cara</string>
     </attribute>
     <widget class="QComboBox" name="cmbTipoCara">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimumContentsLength">
       <number>0</number>
      </property>
      <item>
       <property name="text">
        <string>Redonda</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Rectangular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ovalada</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnAniadir">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitar</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>175</width>
        <height>74</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="lblCara1">
         <property name="text">
          <string>Radio: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="txtCara1"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="txtCara2"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblCara2">
         <property name="text">
          <string>Alto:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pbColor">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Línea</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblColor">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pbColor_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Piel</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblFondo">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Rasgos</string>
     </attribute>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Ojos1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ojos2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ojos3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ojos4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ojos5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ojos6</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="btnOjos">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>210</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitar</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>131</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Posición X:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="spinOjosX">
         <property name="maximum">
          <number>600</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Posición Y:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="spinOjosY">
         <property name="maximum">
          <number>600</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QSpinBox" name="spinOjosW">
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="spinOjosH">
         <property name="maximum">
          <number>300</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Ancho:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Alto:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="cmbRasgos">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Ojos</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nariz</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Boca</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cabello</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>511</width>
      <height>411</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="pbLimpiar">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>340</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Limpiar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGuardarImagen">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>340</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>796</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
